tableauserverclient/server/endpoint/endpoint.py

```diff
@@ -8,64 +8,61 @@
 
 
 class Endpoint(object):
-    def __init__(self):
-        self.parent_srv = None
+    def __init__(self, parent_srv):
+        self.parent_srv = parent_srv
 
     @staticmethod
-    def _check_status(server_response):
-        if server_response.status_code not in Success_codes:
-            raise ServerResponseError.from_response(server_response.content)
+    def _make_headers(token, content_type):
```

Comment:
`_make_tableau_headers` or `_make_server_headers` or something to indicate it's not a totally general purpose function, it really is Tableau REST API specific headers